src/Akka.HealthCheck.Persistence/AkkaPersistenceLivenessProbe.cs

```diff
@@ -178,7 +187,7 @@ private bool HandleMessages(object message)
                         _log.Debug("Recreating persistence probe.");
                     
                     Timers.StartSingleTimer(TimeoutTimerKey, CheckTimeout.Instance, _timeout);
-                    _probe = Context.ActorOf(Props.Create(() => new SuicideProbe(Self, _probeCounter == 0, _id)));
+                    _probe = Context.ActorOf(Props.Create(() => new SuicideProbe(Self, _probeCounter <= _firstIndex, _id, _logInfo)));
```

Comment:
Help me understand the change in the boolean logic here

Reply:
Before, only the first ever probe (probe counter 0) is considered as a warmup cycle, now we extend the first index if the first cycle fails.

Reply:
Got it